synapse/datamodel.py

```diff
@@ -481,7 +481,9 @@ def __init__(self):
         item = s_types.Str(self, 'str', info, {})
         self.addBaseType(item)
 
-        info = {'doc': 'The base hex type.'}
+        info = {
+            'doc': 'The base hex type. ``zeropad`` will be truncated to ``size`` if ``size`` is greater than zero.'
```

Comment:
I don't think we typically document options within the type def. Would get verbose/messy.